src/components/BookUi.jsx

```diff
@@ -1,9 +1,18 @@
+// eslint-disable-next-line react/prop-types
```

Comment:
- [OPTIONAL ] Same here, Please make sure you fix all linter errors instead of disabling them, A linter is great for identifying errors when you use standard rules. Remember, a linter analyzes your code for stylistic and programming errors against the rules it knows. 💯 😉